webb_ai/README.md

```diff
@@ -0,0 +1,51 @@
+# Webb.ai
+
+## Overview
+
+Matt by Webb.ai is the first AI-enabled reliability engineer. Matt troubleshoots Monitors from Datadog, incidents from infrastructure like Kubernetes, and alerts from cloud providers like AWS, Azure, and Google Cloud. It automates troubleshooting to identify the root cause of an alert or incident in less than 5 mins.
+
+With this integration
+- Datadog monitors in alert state will be identified and automatically analyzed by Matt
+- Troubleshooting results will improve as Matt can query Datadog metrics when required(even for kubernetes incidents, aws alerts etc.)
+
+Data queried by Matt:
+- Event stream to identify Monitors in alert state
```

Comment:
Please include what data is specifically sent to datadog/what data is being monitored in Datadog and the value